backoffice/workflows/airflow_utils.py

```diff
@@ -0,0 +1,36 @@
+from os import environ
+
+import requests
+from django.http import JsonResponse
+from requests.exceptions import RequestException
+from rest_framework import status
+
+AIRFLOW_BASE_URL = environ.get("AIRFLOW_BASE_URL")
+
+AIRFLOW_HEADERS = {"Content-Type": "application/json", "Authorization": f"Basic {environ.get('AIRFLOW_TOKEN')}"}
+
+
+def trigger_airflow_dag(dag_id, workflow_id, extra_data=None):
+    """
+    Triggers an airflow dag
+
+    :param dag_id: name of the dag to run
+    :param workflow_id: id of the workflow being triggered
+    :returns: request response
+    :raises RequestException: raises exception on request failure
```

Comment:
actually this is not true, it doesn't raise since we are catching the exception.